adobe/react-spectrum
# fix(#2468): Combobox: onKeyDown event will now be triggered per docs

Closes [2468](https://github.com/adobe/react-spectrum/issues/2468)

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Use allowsCustomValue: true storybook example
2. Type a "pepe le pew" in comboBox
3. Press Enter 

Expected: onKeyDown event fired in the Actions tab

## 🧢 Your Project:
